.github/CODEOWNERS

```diff
@@ -206,6 +206,47 @@
 /tactics/class_tactics.* @coq/typeclasses-maintainers
 /test-suite/typeclasses/ @coq/typeclasses-maintainers
 
+########## Number ##########
+
+/interp/numTok.*                     @coq/number-maintainers
+/kernel/float64*                     @coq/number-maintainers
+/kernel/uint63*                      @coq/number-maintainers
+/plugins/syntax/g_number_string.mlg  @coq/number-maintainers
+/plugins/syntax/int63_syntax_plugin.mllib  @coq/number-maintainers
+/plugins/syntax/number.ml            @coq/number-maintainers
+/plugins/syntax/number_string_notation_plugin.mllib  @coq/number-maintainers
+/user-contrib/Ltac2/Int.v            @coq/number-maintainers
+/test-suite/output/FloatExtraction.v  @coq/number-maintainers
+/test-suite/output/FloatSyntax.v     @coq/number-maintainers
+/test-suite/output/Int31Syntax.v     @coq/number-maintainers
+/test-suite/output/Int63Syntax.v     @coq/number-maintainers
+/test-suite/output/NSyntax.v         @coq/number-maintainers
+/test-suite/output/NumberNotations.v  @coq/number-maintainers
+/test-suite/output/RealSyntax.v      @coq/number-maintainers
+/test-suite/output/sint63Notation.v  @coq/number-maintainers
+/test-suite/output/Sint63Syntax.v    @coq/number-maintainers
+/test-suite/output/ZSyntax.v         @coq/number-maintainers
```

Comment:
Would it make sense to rename these tests with a pattern to avoid having to explicitly list all the files?

Reply:
Indeed